src/desktop/apps/article/routes.ts

```diff
@@ -7,32 +7,20 @@ import {
   SuperSubArticlesQuery,
   SuperArticleQuery,
 } from "desktop/apps/article/queries/superArticle"
-import { positronql as _positronql } from "desktop/lib/positronql"
-import { data as _sd } from "sharify"
-import { stitch as _stitch } from "@artsy/stitch"
-import { getCurrentUnixTimestamp } from "reaction/Components/Publishing/Constants"
+import { positronql } from "desktop/lib/positronql"
+import { data as sd } from "sharify"
+import { stitch } from "@artsy/stitch"
+import { getCurrentUnixTimestamp } from "@artsy/reaction/dist/Components/Publishing/Constants"
```

Comment:
@eessex - if we're going to undo aliasing, might as well update everything at once.

Reply:
Changed this one so that the imports in this file are consistent, but yes I think we should remove these overall because in `ts` files they break typings when linked.